conda_libmamba_solver/solver.py

```diff
@@ -879,21 +881,33 @@ def _package_record_from_json_payload(
         json_payload: str
             A str-encoded JSON payload with the PackageRecord kwargs.
         """
+        kwargs = json.loads(json_payload)
```

Comment:
Any risk here for JSON errors that we should preemptively catch?

Reply:
That would only happen in libmamba fails to create a JSON string and at that point I think we are in deeper trouble. If you want us to raise a `Conda*` error to avoid the ugly traceback, I can do so and I think it's a good idea, but there's no way to workaround such a problem.